### ISSUE TITLE
[Bug]:  Ensures all ARIA attributes have valid values for Themes button

### ISSUE THREAD

**Issue by msftedad** [storybookjs/storybook#24074]
### Describe the bug

 Ensures all ARIA attributes have valid values for Themes button.

**Actual Result:**
All ARIA attributes doesn't have valid values for Themes button.

**Expected Result:**
Ensures all ARIA attributes have valid values for Themes button.

### To Reproduce

1. Open URL: [Common Navigation - Scrolling Panel ⋅ Storybook (windows.net)](https://vivacommonstorybooktest.blob.core.windows.net/$web/index.html?path=/story/common-navigation--scrolling-panel&globals=backgrounds.grid:!false;backgrounds.value:transparent)
2. Navigate to “Themes” button at left side of the pane invoke it. 
3. Now apply Accessibility insights for web and observe the issue at Themes button.

![Themes button](https://github.com/storybookjs/storybook/assets/93735775/9a1b879f-000f-466b-90bb-2ab31525a9fb)
![Themes button2](https://github.com/storybookjs/storybook/assets/93735775/db11de48-e024-42ed-a0c9-7f3482122ea4)

### System

```bash
OS: Windows 11
OS version: 22H2 (OS Build 22621.2134)
Edge Version: 116.0.1938.69 (Official build) (64-bit)
```

### Additional context

NA

**Comment by pedrosoc** [comment]
@shilman, Can you please assign this issue to me?

I'm Brazilian, and I'm currently taking a university class with the goal of promoting open-source projects. I'm a regular user of `storybook`, and I'm highly interested in resolving this issue to contribute to the advancement of the framework!

Please add `hacktoberfest` label if it's possible :smile:

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by Sidnioulz** [comment]
@pedrosoc general heads up, you can work on any issue whenever you want. We don't usually use assignments.

Closing this as a duplicate of https://github.com/storybookjs/storybook/issues/31382 and https://github.com/storybookjs/storybook/issues/31267. I will be working on https://github.com/storybookjs/storybook/issues/31267 this month.
